Tests/AppTests/WebpageSnapshotTests.swift

```diff
@@ -1,20 +1,52 @@
 @testable import App
 
 import SnapshotTesting
+import Vapor
 import XCTest
 
 
 let defaultSize = CGSize(width: 800, height: 600)
 
 class WebpageSnapshotTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        Current.date = { Date(timeIntervalSince1970: 0) }
+    }
+
     func test_home() throws {
-        // record = true
-        assertSnapshot(matching: PublicPage.admin().render(indentedBy: .spaces(2)), as: .lines)
+        let page = PublicPage.admin()
+
+        assertSnapshot(matching: page.render(indentedBy: .spaces(2)), as: .lines)
+
+        #if os(macOS)
+        if !isRunningInCI {
+            assertSnapshot(matching: page, as: .image(size: defaultSize, baseURL: baseURL()))
+        }
+        #endif
+    }
+
+    func test_PackageShowView() throws {
```

Comment:
I've added this snapshot mainly for my own testing. It does pass but might be annoying for you while working on the page so maybe you'd prefer if we gate this with an `XCTSkip` on CI until it's settled down a bit?

Reply:
Yea I deliberately didn't include a snapshot with my work so far as it was just going to break with every commit at the moment. It's fine to include it from now on as that HTML is largely done now. It's mainly the CSS that needs work now.